tests/Translator/Extractor/AdminExtractorTest.php

```diff
@@ -121,4 +121,18 @@ public function testExtractCallsBreadcrumbs(): void
 
         $this->adminExtractor->extract([], $catalogue);
     }
+
+    public function testExtractSetsSubject(): void
```

Comment:
I saw this test and I thought that was the requested one @jorrit . Should we revert?

Reply:
No, don't worry, I added one in https://github.com/sonata-project/SonataAdminBundle/pull/6237